_locales/en/messages.json

```diff
@@ -162,14 +162,29 @@
     "message": "Scratch Addons theme"
   },
   "scratchAddonsThemeDescription": {
-    "message": "This allows you to switch between light and dark themes for this settings page, and the extension's popup."
+    "message": "Choose the style of the Scratch Addons page to light or dark. You can set the mode to automatic options as change at specific time, or to be the same as in your system settings. Automatic settings can also turn on and off add-ons like \"Editor Dark Mode\" and \"Dark WWW\"."
```

Comment:
Might want to say "Website Dark Mode" instead of "Dark WWW", or reword this part to simply say "dark-mode addons", "customizable-colors addons" or something else.